### DISCUSSION TITLE
v.3.4.0

### DISCUSSION THREAD

**Discussion by rdmueller** [docToolchain/docToolchain#1418] [General]
### fixed

* [Build suddenly fails (since changes to Gradle plugin portal)](https://github.com/docToolchain/docToolchain/issues/1414)

### added
* Collapsible left menu (see [example](https://doctoolchain.org/docToolchain/v2.0.x/030_news/index.html))
* [exportEA: Add additional option to set the image format](https://github.com/docToolchain/docToolchain/pull/1389)
* [#1391: Update collectIncludes.gradle](https://github.com/docToolchain/docToolchain/pull/1391)

### changed

* removed link to plausible.io on docs site

<hr /><em>This discussion was created from the release <a href='https://github.com/docToolchain/docToolchain/releases/tag/v.3.4.0'>v.3.4.0</a>.</em>

**Comment by KeuleDurden** [comment]
Seems there is a little typo. I think it should be v3.4.0 instead v.3.4.0. Therefore curl fails when running ./dtcw install doctoolchain.

**Reply by rdmueller** [reply]
yeah, good point. I think I have already fixed this. Can you confirm?

**Reply by KeuleDurden** [reply]
Yes, it's working fine now. Thank you!
